x/auth/ante/sigverify.go

```diff
@@ -346,7 +350,8 @@ func DefaultSigVerificationGasConsumer(
 
 // ConsumeMultisignatureVerificationGas consumes gas from a GasMeter for verifying a multisig pubkey signature
 func ConsumeMultisignatureVerificationGas(
-	meter sdk.GasMeter, sig *signing.MultiSignatureData, pubkey multisig.PubKey, params types.Params,
+	meter sdk.GasMeter, sig *signing.MultiSignatureData, pubkey multisig.PubKey,
+	params types.Params, sigV2 signing.SignatureV2,
```

Comment:
although that doesn't have any real impact, that might be clearer to give `sigV2` parameter a different name than the variable below: https://github.com/cosmos/cosmos-sdk/blob/2dd868bfca3bc00f4c887f673ec3539c521184b2/x/auth/ante/sigverify.go#L364 or the other way around?